platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/command/AbstractProfileCommandTest.java

```diff
@@ -1,104 +0,0 @@
-/**
```

Comment:
❗️ I don't think we can remove this one. It is required for the profile install and profile list commands.